src/openstudio_app/OpenStudioApp.cpp

```diff
@@ -1158,7 +1158,8 @@ void OpenStudioApp::readSettings() {
   setLastPath(settings.value("lastPath", QDir::homePath()).toString());
   setDviewPath(openstudio::toPath(settings.value("dviewPath", "").toString()));
   m_currLang = settings.value("language", "en").toString();
-  LOG_FREE(Debug, "OpenStudioApp", "\n\n\nm_currLang=[" << m_currLang.toStdString() << "]\n\n\n");
+  m_useLabsCLI = settings.value("useLabsCLI", false).toBool();
```

Comment:
Probably want to default to using the new CLI and provide option to use the old one?  For measure manager I just assumed we'd use the new one, do we really need to support the old CLI for measure manager or run purposes?

Reply:
I'm not sure. Perhaps yes, we enable the default of OS SDK.
But yeah, we probably want the fallback to classic for both MeasureManager and run.